app/classifier/classifier.py

```diff
@@ -293,6 +301,51 @@ def learn(self, load_from_file, symbol=""):
             if not load_from_file:
                 self.save_training_set(symbol)
             self.learn_one_symbol(symbol)
+        else:
+            for sym in self.symbol_list:
+                print("learning", sym, "symbol...")
+                self.learn_one_symbol(sym) 
+
+        print("learning all together...")
+        self.learn_all_symbols_together()
+
+    def delete_symbol(self, symbol):
```

Comment:
- [ ] If it is a private function you can use name it `_delete_symbol` - pylint pay less attention to functions named like this.